src/plugins/custom_integrations/common/index.ts

```diff
@@ -51,6 +51,13 @@ export const INTEGRATION_CATEGORY_DISPLAY = {
   geo: 'Geo',
 };
 
+// featured integrations will be brought to the top of the search results for
+// a given category. Integrations are displayed in the order of the array.
+export const FEATURED_INTEGRATIONS_BY_CATEGORY = {
```

Comment:
I thought it best to locate the featured integrations config next to where we define the categories. 

We could look to make this dynamic in the future.